src/detail-page/DetailPage.tsx

```diff
@@ -6,15 +6,14 @@ import { MainContainer } from '../layout/components/MainContainer';
 import { Tabs } from './components/tabs/Tabs';
 
 import { ITProfessions } from '../data/ITProfessions';
+import { ProfessionCard } from '../home-page/components/ProfessionCard';
```

Comment:
We use ProfessionCard in two views now - HomePage and DetailPage. Should we move ProfessionCard outside of HomePage or we keep it there?

Reply:
I moved ProfessionCard to shared directory.